react/src/containers/ParkShow.js

```diff
@@ -30,20 +31,28 @@ class ParkShow extends React.Component {
 
   render() {
     let ratings
+    let park_name
```

Comment:
We use camel case when defining React variables. Please change all instances of `park_name` to `parkName`